tests/scripts/set-release-ver.sh

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env bash
+set -euo pipefail
+
+FROM_TAG=$(grep "docker.io/rook/ceph" deploy/examples/images.txt | awk -F : '{ print $2 }')
+TO_TAG="$1"
```

Comment:
Putting input args at the top serves to make them more self documenting. Because FROM_TAG is no longer an input arg (but still critical to understanding the function), I would suggest making it the second section. A comment also wouldn't hurt

```suggestion
TO_TAG="$1"

FROM_TAG=$(grep "docker.io/rook/ceph" deploy/examples/images.txt | awk -F : '{ print $2 }')

```